database/postgresconn/postgresconn.go

```diff
@@ -405,10 +421,117 @@ func (p *Postgres) ensureVersionTable() (err error) {
 		return nil
 	}
 
-	query = `CREATE TABLE IF NOT EXISTS ` + pq.QuoteIdentifier(p.config.migrationsSchemaName) + `.` + pq.QuoteIdentifier(p.config.migrationsTableName) + ` (version bigint not null primary key, dirty boolean not null)`
+	query = fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %q.%q`+
+		` (version bigint not null, dirty boolean not null)`,
+		p.config.migrationsSchemaName, p.config.migrationsTableName)
 	if _, err = p.conn.ExecContext(context.Background(), query); err != nil {
 		return &database.Error{OrigErr: err, Query: []byte(query)}
 	}
 
+	// add the created_at and info columns to track history and failures of migrations
+	query = fmt.Sprintf(`ALTER TABLE %q.%q `+
+		`ADD COLUMN IF NOT EXISTS created_at timestamp with time zone NULL, `+
```

Comment:
Is this supposed to be updated_at ?... if not, can we have a column for that as well?

Reply:
I  don't think we ever would update a migration, not seeing how updated_at would be used...